### PR TITLE
Bookstore structure

### PR DESCRIPTION
**Pull request by Alusp**
### I implemented the following in this PR:

- [ ] Use Gitflow.
- [ ] Add Linters.
- [ ] Initialize React app.
- [ ] The building blocks of your app should be set as re-usable components. Create a directory for your components.
- [ ] Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two Routes and Links for the app's navigation:
- [ ] Books - the default view:
- [ ] Display empty list of books ready for the data with the Remove button (no funcionality yet).
- [ ] Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
- [ ] Should have a form for adding a book (no functionality yet)
- [ ] Create a separate component for this form with inputs for a title and an author
- [ ] Add basic styling and disabled it for future use.

### CODE REVIEW

File: src/pages/Author.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Author() {
+  return (
+    <div>
+      <h1>Authors</h1>
```
Comment (victorokoroji) [inline src/pages/Author.js]: - [x]  Kindly note that the project requirement expects you to display only a button with the text "Check status" on the categories page. Please correct that 😄

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Book({ title }) {
```
Comment (victorokoroji) [inline src/components/Books.js]: - [x] This component should take the title and the author as props, not just the title. Kindly fix that 👍🏽

File: src/components/NavigationBar.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { FaUser } from 'react-icons/fa';
+
+function NavigationBar() {
+  return (
+    <nav className="py-6 shadow-md">
+      <div className=" items-center bg-blue-500 text-white p-5">
+        <h1 className="text-2xl font-semibold">Bookstore CMS</h1>
+        <div className="flex">
+          <Link to="/" className="flex-none w-14 h-14 text-xl font-semibold mx-10 ">Books</Link>
+          <Link to="/author" className=" flex-initial w-64 text-xl font-semibold mx-2">Author</Link>
```
Comment (victorokoroji) [inline src/components/NavigationBar.js]: - [x] I noticed that in your <App /> component, you are routing the categories page to` /categories` path i.e.`<Route path="/categories" element={<Categories />} />` where as you are linking to /author in your Navbar which means the incorrect routing. Please correct that 👍🏽

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @Alusp, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️

- [x] Good job creating the react app 👍🏽 . Kindly note that the project requirement expects you to have a categories page, which means your component should be `<Categories />` and not `<Author />`. Please correct that 😄 

- [x] I noticed that you have two <Books /> component in your app which serves a different purpose. I think it will be better if each component has there unique name to avoid any form of confusion for developers who would like to use or maintain your app. 

The component that takes the book data as a prop and renders the title and the author should be named as `<Book />` component not `<Books />`

_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alusp** [comment]
Thanks very much for the review @vickymarz, Am about to fix the required changes.

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Alusp 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alusp** [comment]
**Thanks very much for the review** @AnselemOdims

**Pull request merged**